packages/rpc/test/chainStorageWatcher.test.ts

```diff
@@ -110,6 +110,29 @@ describe('makeAgoricChainStorageWatcher', () => {
     expect(await value).toEqual(expected);
   });
 
+  it('can do single queries', async () => {
+    const expected = 126560000000;
+    const path = 'published.vitest.unserializedValue';
```

Comment:
Important: is `published` necessary? if not remove it and if so explain in a comment

Reply:
No just to illustrate, removed